src/pages/index.tsx

```diff
@@ -1,23 +1,45 @@
 import * as React from "react";
 import type { HeadFC, PageProps } from "gatsby";
 import { Container } from "../components/Container/Container";
-import { Link } from "nerdux-ui-system";
+import { Link, Checkbox, TextField, Switch, Button } from "nerdux-ui-system";
 
 import * as styles from "./index.module.scss";
 
 const IndexPage: React.FC<PageProps> = () => {
   return (
     <main>
       <Container>
-        <header className={styles.titleHeader}>
-          <h1>Lead generator project</h1>
-          <Link
+        <div className={styles.sidebarleft}></div>
```

Comment:
**suggestion:** I would wrap it all inside one `div` that would be treated as a layout component